easybuild/toolchains/crayintel.py

```diff
@@ -38,3 +38,4 @@ class CrayIntel(CrayPEIntel, CrayMPICH, LibSci):
     """Compiler toolchain for Cray Programming Environment for Intel compilers (PrgEnv-intel)."""
     NAME = 'CrayIntel'
     SUBTOOLCHAIN = DUMMY_TOOLCHAIN_NAME
+    CrayPEIntel.COMPILER_UNIQUE_OPTION_MAP.update({'openmp': 'openmp'})
```

Comment:
`openmp` is deprecated for recent Intel versions, better stick to `fopenmp` here too, cfr. #1718